fl4health/losses/mkmmd_loss.py

```diff
@@ -0,0 +1,177 @@
+# multi-kernel maximum mean discrepancy
+# cao bin, HKUST, China, binjacobcao@gmail.com
+# free to charge for academic communication
+
+from typing import List, Optional
+
+import torch
+from qpth.qp import QPFunction
+
+
+class MKMMDLoss(torch.nn.Module):
+    def __init__(
+        self,
+        device: torch.device,
+        gamma_list: List[float] = [
```

Comment:
Mutable values should not be set as default values for parameters, see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments . Not a big problem since on line 32 you immediately make a tensor out of it but it's good to avoid in case that line is changed in the future.

I'm surprised mypy didn't pick up on it, maybe it's smarter than I think it is?